src/back-end/lib/db.ts

```diff
@@ -1299,6 +1309,11 @@ export const readManyCWUProposals = tryDb<[Id], CWUProposalSlim[]>(async (connec
     proposal.status = statusResult.status;
   }
 
+  // Filter out any proposals not in UNDER_REVIEW or later status if admin/gov owner
+  if (session.user && session.user.type !== UserType.Vendor) {
+    results = results.filter(result => ![CWUProposalStatus.Draft, CWUProposalStatus.Submitted].includes(result.status));
```

Comment:
This proposal status check should be abstracted to a simple helper function in the shared proposal resource since it's being used multiple times.

Reply:
e.g. `isProposalVisibleToGovernment(proposal: CWUProposal): boolean)`